srcs/tokenizer.c

```diff
@@ -52,14 +59,31 @@ int	check_token_literals(char *str)
 	return (-1);
 }
 
+static void	string_decetion(t_token *tok, char **str)
```

Comment:
???

Reply:
yes, well sometimes I make typs not but not oftne